apps/dashboard/src/app/(dashboard)/(chain)/components/server/products.ts

```diff
@@ -37,6 +37,11 @@ export const products = [
     id: "pay",
     icon: PayIcon,
   },
+  {
+    name: "Faucet",
+    id: "faucet",
+    icon: PayIcon,
+  },
```

Comment:
this does not actually get returned from the api, does it?

Reply:
no, it's added here to show the faucet tab in the chain page